cnf-tests/testimages/images.json

```diff
@@ -1,22 +1,12 @@
 [
     {
-        "name": "sctptester",
+        "name": "cnf-tests",
```

Comment:
Following the comment in https://github.com/openshift-kni/cnf-features-deploy/pull/196/files#r422859058 , I would change this too to be registry / image.

For example:
```
{
        "image": "dpdk:4.5",
        "registry": "quay.io/openshift-kni/"
    }
```

Note that this requires a change in the mirror.go file.